.expeditor/update_version.sh

```diff
@@ -7,9 +7,9 @@
 set -evx
 
 sed -i -r "s/^(\s*)VERSION = \".+\"/\1VERSION = \"$(cat VERSION)\"/" components/ruby/lib/license_acceptance/version.rb
-cd components/ruby
-bundle update
-cd ../..
+pushd components/ruby
+bundle update license-acceptance
+popd ../..
```

Comment:
I don't think popd takes a directory argument, just a numeral for number of items to pop

Reply:
the numeral is optional. This should just be `popd`